### PR TITLE
update: add more columns on to read scrape data

### PR DESCRIPTION
**Pull request by nmmarquez**
Have read scrape data return data columns even if they dont appear in the google sheet. `write_scrape_data` in the scraper repo will now do the filtering down process of only selecting the appropriate columns

### CODE REVIEW

File: R/read_scrape_data.R [hunk 1]
```diff
@@ -99,6 +99,7 @@ read_scrape_data <- function(
                 Staff.Recovered, Residents.Tadmin, Staff.Tested, Residents.Negative,
                 Staff.Negative, Residents.Pending, Staff.Pending,
                 Residents.Quarantine, Staff.Quarantine, Residents.Active,
+                Residents.Popultion, Residents.Tested,
```
Comment (hjohns12) [inline R/read_scrape_data.R:102]: Misspelling in `Residents.Population`

**Review by hjohns12 — COMMENTED** [review]
Will this error out if the columns called "Residents.Tested" and "Residents.Population" don't exist?

**Comment by nmmarquez** [comment]
> Will this error out if the columns called "Residents.Tested" and "Residents.Population" don't exist?

No they will still work as those columns exist in the combined data. Also thanks for catching the mis-spelling

**Review by erika-tyagi — APPROVED** [review]
Maybe the first time the CI has been helpful and not hurtful...?? But yay thanks!

**Review by hjohns12 — APPROVED** [review]

**Comment by erika-tyagi** [comment]
Sorry another question: Does it make sense to add the other columns in the new info sheets? (e.g. Description, Security, Age, BJS.ID etc.) - or I guess just removing the select statement? 

It looks like we select the columns for the Google Sheet in `write_latest_data` so adding more stuff here won't break things, but might as well get everything from `read_scrape_data` to avoid a second join with the info sheet?

**Comment by nmmarquez** [comment]
@erika-tyagi yeah that works for me. We will still have a select statement otherwise we get all of the crosswalk inf names as well. Just a heads up that those columns are also returned when debug = T

**Pull request merged**